react/src/views/Pages/Upload.jsx

```diff
@@ -13,8 +62,49 @@ class UploadPage extends React.Component {
     super(props);
 
     this.state = {
-      uploadURL: ""
+      uploadURL: "",
+      checked: [24, 22],
+      selectedValue: null,
+      selectedEnabled: "b",
+      vrtitle:""
     };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleChangeEnabled = this.handleChangeEnabled.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ selectedValue: event.target.value });
+  }
+  handleChangeEnabled(event) {
+    this.setState({ selectedEnabled: event.target.value });
+  }
+  handleToggle(value) {
+    const { checked } = this.state;
+    const currentIndex = checked.indexOf(value);
+    const newChecked = [...checked];
+
+    if (currentIndex === -1) {
+      newChecked.push(value);
+    } else {
+      newChecked.splice(currentIndex, 1);
+    }
+
+    this.setState({
+      checked: newChecked
+    });
+  }
+
+  updateVRTitle = e => {
+    this.setState(
+      {
+        vrtitle: e.target.value
+      })
+  }
+
+  uploadVRVideo()
+  {
+      console.log("Upload the video")
+      console.log("Upload Video Name is: " + this.state.vrtitle)
```

Comment:
I don't see the `vrtitle` being printed out in the console. This is what I get:
![image](https://user-images.githubusercontent.com/36905285/69448290-0f42e680-0d0d-11ea-8c35-0c996add9589.png)